### PR TITLE
Codecs File Created

### PR DESCRIPTION
**Pull request by SanjaySinghRajpoot**
In this file there are instructions on How to download and install Codecs for DVD's and more.

### CODE REVIEW

**Review by why-not-try-calmer — CHANGES_REQUESTED** [review]
Cool first PR @SanjaySinghRajpoot ! 

I just don't quite understand: Why is this about Windows?...

**Comment by SanjaySinghRajpoot** [comment]
ok i will correct it

**Comment by why-not-try-calmer** [comment]
Thank you, I have some work to do, will get back to this in a couple of days.

**Comment by why-not-try-calmer** [comment]
Thank you, now I'd be great to have the instructions for installing through `opi`. Could you please add them?

**Review by why-not-try-calmer — CHANGES_REQUESTED** [review]
Thanks, can you add the `opi` instructions too?

**Comment by SanjaySinghRajpoot** [comment]
ok I will update changes.

**Review by why-not-try-calmer — APPROVED** [review]

**Review by why-not-try-calmer — CHANGES_REQUESTED** [review]
Thanks better, thanks a lot! Some suggestions (only the first two are blockers for me):
- (blocker) please make sure that, for every new "instruction" you give to the user in plain English,  there is a corresponding set of command lines or UI actions ("click this then that then finally click there"), and vice-versa
  - vice-versa means that every command line is appropriately commented (no "orphan" command line with no comment)
- (blocker) the `opi` alternative should be mentioned much earlier in the text, since it's really an alternative to other ways of installing codecs
- (optional) please add a short introduction / explanation of why openSUSE does not ship with codecs pre-installed

**Review by apinter — COMMENTED** [review]
Thank you for contributing to the openSUSE-docs, appreciated.
A few comments I would add:
* It is good practice to set a priority with repos being added to the system. In case of Packman the recommended priority is 90 that would require a change in line 32 like you added in line 22 and 26.
* You mention YaST installer, don't forget to add the instructions. 
* IMO if you add Packman already, adding the VLC repo feels a bit redundant.

**Comment by why-not-try-calmer** [comment]
> Thank you for contributing to the openSUSE-docs, appreciated.
> A few comments I would add:
> 
> * It is good practice to set a priority with repos being added to the system. In case of Packman the recommended priority is 90 that would require a change in line 32 like you added in line 22 and 26.
> * You mention YaST installer, don't forget to add the instructions.
> * IMO if you add Packman already, adding the VLC repo feels a bit redundant.

On the VLC redundancy: it's kinda my fault, not @SanjaySinghRajpoot's. The idea was: if someone just needs to play a video from time to time, they might not need the full Packman essentials set, so it might be nice of us to provide a quick escape route for these users.

**Comment by surprized** [comment]
Here is this wiki article I have been maintaining, may help 
https://en.opensuse.org/SDB:Installing_codecs_from_Packman_repositories

**Comment by why-not-try-calmer** [comment]
Any news @SanjaySinghRajpoot ?
